rfcs/0003-languages.md

```diff
@@ -0,0 +1,159 @@
+---
+feature: (languages)
+start-date: (2021-04-30)
+author: (Shahrukh Khan)
+related-issues: (None)
```

Comment:
```suggestion
start-date: 2021-04-30
author: Shahrukh Khan
related-issues: None
```
For related issues and PRs, there are the previous RFCs about themes and mkSoxinModule